spectrum.js

```diff
@@ -136,7 +136,9 @@
             var current = p[i];
             if(current) {
                 var tiny = tinycolor(current);
-                var c = tiny.toHsl().l < 0.5 ? "sp-thumb-el sp-thumb-dark" : "sp-thumb-el sp-thumb-light";
+                var hsl = tiny.toHsl();
+                var darkColor = hsl.l + (1 - hsl.a);
+                var c = darkColor < 1.0 ? "sp-thumb-el sp-thumb-dark" : "sp-thumb-el sp-thumb-light";
```

Comment:
In the case of alpha = 1 it looks like this would change the formula to be checking for l < 1, which would be every color, right?